playbooks/support/responding.md

```diff
@@ -11,6 +11,23 @@ respond to these issues.
   These automated system alerts _may_ be the first sign of a problem but don't _necessarily_ warrant treatment as
   immediate incidents. E.g., load-balancers may route around failing EC2 instances or auto-scaling may bring up
   capacity in response to load.
+- Monitor and acknowledge automated uptime incidents created by OpsGenie.
+
+  - For high-criticality systems, OpsGenie will open incidents if
+    [Pingdom](https://my.pingdom.com/app/newchecks/checks) uptime checks are failing. Alerts will automatically
+    close if resolved.
+  - All engineers should have the OpsGenie app for
+    [iPhone](http://itunes.apple.com/us/app/opsgenie/id528590328?mt=8) or
+    [Android](https://play.google.com/store/apps/details?id=com.ifountain.opsgenie&hl=en) installed and be logged
+    in.
+  - Select your preferences for push notifications, text messages, and/or phone calls for alerting in your
+    [OpsGenie user notification settings](https://artsy.app.opsgenie.com/settings/user/notification).
+
+    Recommended settings:
+
+    - Immediately via push notification
+    - After 5 minutes via SMS
+    - After 15 minutes via phone call
```

Comment:
Currently, I don't think responders have the ability to customize these notification preferences. We're setting that up globally via a Central Notification Template:

![image](https://user-images.githubusercontent.com/123595/77459893-bc6e5e00-6dd6-11ea-996e-88164e3db574.png)

https://docs.opsgenie.com/docs/central-notification-templates
https://artsy.app.opsgenie.com/settings/notificationSettings/detail/5ebe8a41-9752-4b75-8bd6-a16d52824759

I think the reason for setting this up originally was to ease on-boarding. New on-call members wouldn't have to set up their own notification preferences. But if we'd rather require on-call responders to set these up before shifts, then we could certainly consider removing the global override. There's also the question of whether having consistent notification rules across the team is more valuable than any single member being able to customize each notifcation methods and thresholds.

Reply:
Added this at the suggestion of @izakp but I'm happy to just let everyone use the defaults.